### PR TITLE
Update ui with javadocs

### PR DESCRIPTION
**Pull request by seanjyjy**
Pr Overview
* Update java docs for all UI classes
* Reformat some code in UI.

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -288,28 +304,28 @@ public void changeListPanelView(TabName tabName) {
...
     /**
-     * todo javadocs
+     * @return An Optional value containing the profile list panel.
      */
     private Optional<ListPanel<? extends Item>> setProfileTabView() {
         return Optional.of(new ProfileListPanel(profileItems));
     }
 
     /**
-     * todo javadocs
```
Comment (orzymandias) [inline src/main/java/seedu/address/ui/MainWindow.java:312]: Javadoc could perhaps start with Switches.... depending on

File: src/main/java/seedu/address/ui/cards/LineNumber.java [hunk 2]
```diff
@@ -1,5 +1,8 @@
 package seedu.address.ui.cards;
 
+/**
```
Comment (orzymandias) [inline src/main/java/seedu/address/ui/cards/LineNumber.java:3]: Perhaps you could change it to Represents a class...

File: src/main/java/seedu/address/ui/display/ProfileDisplay.java [hunk 3]
```diff
@@ -26,34 +48,37 @@ private ProfileDisplay(Stage primaryStage, ProfileItem profileItem) {
...
+     * Set the title of the {@code profileItem} for display.
      */
     private void setInformationTitle() {
         Object jobTitle = mapping.get(TITLE_DISPLAY_NAME);
         setInformationTitle(jobTitle.toString());
     }
 
     /**
-     * todo javadocs
+     * Set the information of the {@code profileItem} for display.
```
Comment (orzymandias) [inline src/main/java/seedu/address/ui/display/ProfileDisplay.java]: Start with sets..

**Review by orzymandias — COMMENTED** [review]
Looks good, just some minor nits with Javadocs to comply with coding standard

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (seanjyjy) [reply src/main/java/seedu/address/ui/MainWindow.java:312]: resolved in the latest push.

File: src/main/java/seedu/address/ui/cards/LineNumber.java (hunk 2 above)
Comment (seanjyjy) [reply src/main/java/seedu/address/ui/cards/LineNumber.java:3]: resolved in the latest push.

File: src/main/java/seedu/address/ui/display/ProfileDisplay.java (hunk 3 above)
Comment (seanjyjy) [reply src/main/java/seedu/address/ui/display/ProfileDisplay.java]: resolved in the latest push.

**Review by keanecjy — APPROVED** [review]
LGTM

**Pull request merged**